CHANGES.md

```diff
@@ -0,0 +1,17 @@
+# Changes
+
+## [Unreleased](https://github.com/crim-ca/stac-populator) (latest)
+
+* Add `LICENSE` file.
+* Add `bump-my-version` with `make version` and `make VERSION=<...> bump` utilities to self-update release versions.
+* Add more metadata to `pyproject.toml`.
+* Adjust `README.md` with updated references and release version indicators.
+* Add `CHANGES.md` to record version updates.
+* Add `dev` dependencies to `pyproject.toml` for testing the package (install with `pip install ".[dev]"`).
+* Add GitHub CI tests.
+* Remove `requirements.txt` in favor of all dependencies combined in `pyproject.toml`.
+* Refactor of `CMIP6_UofT` with more robust parsing strategies and STAC Item generation from THREDDS NCML metadata.
```

Comment:
This bullet point seems not relevant to this PR.

Reply:
Yes. This is because the Changelog did not already exist. It would be present following the merge of yesterday. I can separate it from this PR into distinct versions to make it clearer.

Reply:
Done in https://github.com/crim-ca/stac-populator/pull/28/commits/40c34cdc896fae2c84fa056ea4ac254c41720a4c